### PR TITLE
Adding medium scale ROSA for 4.14

### PR DESCRIPTION
**Pull request by krishvoor**
This PR introduces adding medium scale ROSA testing

### CODE REVIEW

**Comment by jtaleric** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-nightly-perfscale-rosa-multiaz-cluster-density-v2-120nodes

**Comment by jtaleric** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-nightly-perfscale-rosa-multiaz-cluster-density-v2-120nodes

**Comment by jtaleric** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-nightly-perfscale-rosa-multiaz-cluster-density-v2-120nodes

**Comment by jtaleric** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-nightly-perfscale-rosa-multiaz-cluster-density-v2-120nodes

**Comment by jtaleric** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-nightly-perfscale-rosa-multiaz-cluster-density-v2-120nodes

**Comment by jtaleric** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-nightly-perfscale-rosa-multiaz-cluster-density-v2-120nodes

**Comment by jtaleric** [comment]
@krishvoor so, the PR calls out cluster-density-v2, but it was running both cluster-density-v2 and node-density. I have updated to just cdv2

**Comment by jtaleric** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-nightly-perfscale-rosa-multiaz-cluster-density-v2-120nodes

**Comment by krishvoor** [comment]
> @krishvoor before we merge lets discuss if we want to have both cdv2 and node-density run at this scale.

@jtaleric Updated, ptal!

**Comment by jtaleric** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-nightly-perfscale-rosa-multiaz-ci-tests-medium-scale-kube-burner

**Comment by krishvoor** [comment]
The error encountered is not the PR in itself.
But, [due to the lack of privileges](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/41979/rehearse-41979-periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-nightly-perfscale-rosa-multiaz-ci-tests-medium-scale-kube-burner/1689703436863410176/artifacts/perfscale-rosa-multiaz-ci-tests-medium-scale-kube-burner/gather-audit-logs/build-log.txt) to delete a secret in gather-audit-logs secrets.

**Comment by krishvoor** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-nightly-perfscale-rosa-multiaz-ci-tests-medium-scale-kube-burner

**Comment by krishvoor** [comment]
The error observed is during [cluster-operator-wait-status](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/41979/rehearse-41979-periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-nightly-perfscale-rosa-multiaz-ci-tests-medium-scale-kube-burner/1689842287535722496)

The default timeout is 30 mins [per this](https://steps.ci.openshift.org/reference/rosa-cluster-wait-ready-operators)

**Comment by jtaleric** [comment]
Another `cluster-wait-ready-operators` failure :/

**Comment by jtaleric** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-nightly-perfscale-rosa-multiaz-ci-tests-medium-scale-kube-burner

**Comment by jtaleric** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-nightly-perfscale-rosa-multiaz-ci-tests-medium-scale-kube-burner

**Comment by jtaleric** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-nightly-perfscale-rosa-multiaz-ci-tests-medium-scale-kube-burner

**Comment by jtaleric** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-nightly-perfscale-rosa-multiaz-ci-tests-medium-scale-kube-burner

**Comment by jtaleric** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-nightly-perfscale-rosa-multiaz-ci-tests-medium-scale-kube-burner

**Comment by jtaleric** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-nightly-perfscale-rosa-multiaz-ci-tests-medium-scale-kube-burner

**Comment by jtaleric** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-nightly-perfscale-rosa-multiaz-ci-tests-medium-scale-kube-burner

**Comment by jtaleric** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-nightly-perfscale-rosa-multiaz-ci-tests-medium-scale-kube-burner

**Comment by jtaleric** [comment]
@krishvoor please rebase.

**Comment by jtaleric** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-nightly-perfscale-rosa-multiaz-ci-tests-medium-scale-kube-burner

**Comment by jtaleric** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-nightly-perfscale-rosa-multiaz-ci-tests-medium-scale-kube-burner

**Comment by jtaleric** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-nightly-perfscale-rosa-multiaz-ci-tests-medium-scale-kube-burner

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.14-nightly.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: perfscale-rosa-multiaz-ci-tests-medium-scale-kube-burner
+  cron: 0 12 * * 6
+  steps:
+    cluster_profile: aws-perfscale
+    env:
+      CHANNEL_GROUP: nightly
+      COMPUTE_MACHINE_TYPE: m5.xlarge
```
Comment (afcollins) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.14-nightly.yaml:30]: Please try with 2xlarge. Will see if it helps us with cluster-density-v2 results.

**Comment by jtaleric** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-perfscale-rosa-multiaz-ci-tests-medium-scale-kube-burner

**Comment by jtaleric** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-nightly-perfscale-rosa-multiaz-ci-tests-medium-scale-kube-burner

**Comment by krishvoor** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-nightly-perfscale-rosa-multiaz-ci-tests-medium-scale-kube-burner

**Comment by jtaleric** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-nightly-perfscale-rosa-multiaz-ci-tests-medium-scale-kube-burner

**Comment by jtaleric** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-nightly-perfscale-rosa-multiaz-ci-tests-medium-scale-kube-burner

**Comment by dry923** [comment]
+1 to leaving the instance type for the 120 runs at m5.xl and run whatever additional data gathering is necessary for the bug on larger nodes as a one off. We would accept that the current test with m5.xl's will "fail" from a pod latency perspective until the issue is resolved or given a wont fix (at which point we would need to re-evaluate our options).

Additionally, we should investigate what happens if we swap to m6i.xlarge instances to see if that added performance allows it to now "fit" on the smaller instance sizes.

**Review by jtaleric — APPROVED** [review]
lgtm

**Pull request merged**